en/middleware/micrortps.md

```diff
@@ -202,10 +160,11 @@ For example, in order to run the *Client* daemon with SITL connecting to the Age
 micrortps_client start -t UDP
 ```
 
-## Agent in a ROS-independent Offboard Fast RTPS interface
+
+## Agent in a ROS-independent Offboard Fast DDS interface
 
 The *Agent* code is automatically *generated* when you build the associated PX4 firmware.
-You can find the source here: **build/<target-platform>/src/modules/micrortps_bridge/micrortps_client/micrortps_agent/**.
+You can find the source here: `build/<target-platform>/src/modules/micrortps_bridge/micrortps_client/micrortps_agent/`.
```

Comment:
FYI @TSC21 This was why the page wasn't rendering. Vue was seeing the `<target-platform>` as an opening HTML tag. I put backticks around it, changing it to code from Vue perspective. Page now works.

Reply:
Ahah nice catch! Thanks!

Reply:
No worries. FYI, the Vue developer tools can be added as a chrome extension - that then points to this kind of problems in the console.